app/components/round-tracker.js

```diff
@@ -0,0 +1,145 @@
+'use strict'
+import React, { useState, useEffect } from 'react'
+import { createUseStyles } from 'react-jss'
+import StatusBadge from './status-badge'
+import Theme from './theme'
+import cx from 'classnames'
+
+const RoundTracker = ({ roundsStatus = [], className, ...otherProps }) => {
+  const classes = useStyles()
+
+  const [isMobile, setIsMobile] = useState(false)
```

Comment:
In mobile mode, it first renders as 3 wide and then changes to 2 wide after a delay. 
Try this:
```
  const [isMobile, setIsMobile] = useState(
    typeof window !== 'undefined' ? window.matchMedia(`(max-width: ${Theme.condensedWidthBreakPoint})`) : false
  )
```
When this is rendered on the server side, window won't be defined so we need to test for that.

Reply:
There are two issues causing 3 to be rendered first, and then 2 in mobile mode.  The first is that isMobile needs to be set to true, initially so that the component will render 2 initially.  The code above will do that.